Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -799,10 +799,8 @@ def get_attachments(attachments, incident_modified_date, only_new=True):
                 file_results.append(fileResult(filename=filename, data=attachments_zip))
         else:
             for attachment in attachments:
-                attachment_modified_date: datetime = parse(
-                    str(dict_safe_get(attachment, ['created'], "", str))
-                ).replace(tzinfo=pytz.UTC)
-                if incident_modified_date <= attachment_modified_date:
+                attachment_modified_date: datetime = parse(dict_safe_get(attachment, ['created'], "", str))
+                if incident_modified_date < attachment_modified_date:
```

Comment:
not sure about dose changes with removing utc.
lets talk about that and explain time.